doc/faq/crypto-considerations.rst

```diff
@@ -55,21 +55,25 @@ This way a database administrator is not able to inject rogue password hashes.
 
 The admin password hashing is performed in *lib.crypto:hash_with_pepper*.
 
+.. _faq_crypto_audit:
+
 Audit Signing
 ~~~~~~~~~~~~~
 
-The audit log is digitally signed. (see :ref:`Audit` and :ref:`cfgfile`).
+The audit log is digitally signed. (see :ref:`audit` and :ref:`audit_parameters`).
 
-The audit log can be handled by different modules. privacyIDEA comes with an
-SQL Audit Module.
+The audit log can be handled by different modules. privacyIDEA comes with a
```

Comment:
@plettich  "an" is correct!

Reply:
I see. It depends on if SQL is pronounced "esS-Queue-elL" or "sequel".

Reply:
Done in 1ce2fd99b